### DISCUSSION TITLE
The role of sqlitestudiocli.exe

### DISCUSSION THREAD

**Discussion by Mickychen00** [pawelsalawa/sqlitestudio#4404] [Q&A]
### Details
Hi! Is there an introduction about the sqlitestudiocli.exe's function and role? The manual is empty in the Command Line Interface section (https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#command-line-interface-cli)

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
It's still experimental, thus not well documented. But if you run it there is a `.help` command.
